workflows/request_next_topic.ts

```diff
@@ -0,0 +1,56 @@
+import { DefineWorkflow, Schema } from "deno-slack-sdk/mod.ts";
+import { SendRequestToSpeakerFunction } from "../functions/request_next_topic_definition.ts";
+
+/**
+ * A Workflow composed of two steps: asking for details from the user,
+ * and then forward the details along with two buttons (approve and deny) to the selected speaker.
+ */
+export const Request_next_topic = DefineWorkflow({
+  callback_id: "next_topic",
+  title: "Request to Move on to Next Topic",
+  description: "Request to move on to the next topic",
+  input_parameters: {
+    properties: {
+      interactivity: {
+        type: Schema.slack.types.interactivity,
+      },
+    },
+    required: ["interactivity"],
+  },
+});
+
+// Step 1: opening a form for the user to input their request
+const formData = Request_next_topic.addStep(
```

Comment:
not sure if important, but for the CreateMeeting and CreateReminder workflows, we used `SetupWorkflowForm` for this part
I wonder if we should keep it consistent?

Reply:
Thank you for your comments. I'll edit accordingly before class tomorrow